scripts/yyParticle.js

```diff
@@ -1468,6 +1801,27 @@ function	ParticleSystem_Particles_Count( _ps )
 ///				ID of the new particle system
 ///			</returns>
 // #############################################################################################
+/*function	ParticleSystem_Create(_layerID,_persistent)

```

Comment:
Can we delete this?

Reply:
I forgot about this. It's actually supposed to be the other way around - delete the function below and uncomment this.